app/models/weam.rb

```diff
@@ -155,9 +160,14 @@ def approved?
     return false if poo_status.blank? || applicable_law_code.blank?
```

Comment:
Let's clean up these `returns` at the end here.  It's a bit hard to follow:
```suggestion
    return false if applicable_law_code.blank? || applicable_law_code =~ Regexp.new("#{ALC1}|#{ALC2}", 'i')
    return false unless poo_status =~ Regexp.new('aprvd', 'i')
    return vet_tec_flags_for_approved? if applicable_law_code =~ Regexp.new("#{ALCVT}", 'i')
    flags_for_approved?
```
We don't need to check if `poo_status.blank?` because the next check (`poo_status =~ Regexp.new('aprvd', 'i')`) covers it.
Please double check my logic!

Reply:
I think it's ok to leave the returns the way they are as this is more explicit and easier to read for future developers even if they could be cleaned up

Reply:
made a minor tweak to use a single line if statement for VET TEC ONLY checks